util/topsql/stmtstats/stmtstatstest/stmtstats_test.go

```diff
@@ -80,22 +80,35 @@ func TestExecCount(t *testing.T) {
 	})
 
 	// Execute CRUD.
-	const ExecCountPerSQL = 100
-	_, insertSQLDigest := parser.NormalizeDigest("insert into t values (0);")
+	const ExecCountPerSQL = 10
+	var insertSQLDigest, updateSQLDigest, selectSQLDigest, deleteSQLDigest *parser.Digest
 	for n := 0; n < ExecCountPerSQL; n++ {
-		tk.MustExec(fmt.Sprintf("insert into t values (%d);", n))
+		sql := fmt.Sprintf("insert into t values (%d, sleep(0.01));", n)
+		if n == 0 {
+			_, insertSQLDigest = parser.NormalizeDigest(sql)
+		}
+		tk.MustExec(sql)
 	}
-	_, updateSQLDigest := parser.NormalizeDigest("update t set a = 0 where a = 0;")
 	for n := 0; n < ExecCountPerSQL; n++ {
-		tk.MustExec(fmt.Sprintf("update t set a = %d where a = %d;", n, n))
+		sql := fmt.Sprintf("update t set a = %d where a = %d and sleep(0.01);", n, n)
```

Comment:
This sleep duration is only 10ms. It can be very likely that you will get an actual execution duration very different to 10ms and makes your test not effective.

Reply:
Ok. But in my laptop, those SQLs execution duration is around 11ms, and then subtract sleep time by 10ms, the SQL itself only execute about 1ms.  I will enlarge this sleep for machines with different performance.